python_modules/libraries/dagstermill/dagstermill/solids.py

```diff
@@ -98,8 +98,8 @@ def get_papermill_parameters(step_context, inputs, output_log_path):
     check.dict_param(inputs, "inputs", key_type=str)
 
     run_id = step_context.run_id
-
-    marshal_dir = "/tmp/dagstermill/{run_id}/marshal".format(run_id=run_id)
+    temp_dir = tempfile.gettempdir()
```

Comment:
can we use `dagster.seven.temp_dir.get_system_temp_directory` instead here -- on unix variants we prefer this to be under `/tmp` 🙏

Reply:
Yes sure.
It is supposed to do that according to the documentation.

https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

Reply:
I would be just curious why this since it is less general (even if most POSIX system have `/tmp`)